code/obj/item/uplinks.dm

```diff
@@ -36,7 +37,7 @@ Note: Add new traitor items to syndicate_buylist.dm, not here.
 	New()
 		..()
 		SPAWN_DBG(1 SECOND)
-			if (src && istype(src) && (!src.items_general.len && !src.items_job.len && !length(src.items_objective)))
```

Comment:
stop using `.len`

Reply:
removed every instance of .len from uplinks.dm